src/urg_node_driver.cpp

```diff
@@ -535,7 +527,7 @@ void UrgNode::scanThread()
     // service next.
     ros::spinOnce();
 
-    if (!urg_ || !ros::ok)
+    if (!urg_ || !ros::ok())
```

Comment:
Right now, I kept my hands away from the urg_node code (only changed the status request to use the worker instead), as I did not want to change the default driver just so that if the UAM version gives troubles, we have a stable version to fall back to.

But I will have a look if this is a potential problem asap